index.html

```diff
@@ -972,7 +964,7 @@ <h3 property="schema:name skos:prefLabel">Terminology</h3>
                     <dfn id="solid-profile">Solid Profile</dfn>
                   </dt>
                   <dd about="#solid-profile" property="skos:definition">
-                    A <dfn>Solid Profile</dfn> is...
+                    A <dfn>Solid profile</dfn> is a representation of a social agent's identity within the Solid ecosystem, which can be used to describe their persona, preferences, abilities, affiliations, social activities and relationships, characteristics, and digital presence, as well as to aid in the discovery of storage and location of specific data.
```

Comment:
```suggestion
                    A <dfn>Solid profile</dfn> is a description of a social agent's identity within the Solid ecosystem, which can be used to describe their persona, preferences, abilities, affiliations, social activities and relationships, characteristics, and digital presence, as well as to aid in the discovery of storage and location of specific data.
```

I think representation might suggest a specific serialization / media-type